config/clusters/openscapes/workshop.values.yaml

```diff
@@ -80,7 +80,7 @@ basehub:
                     cpu_guarantee: 3.725
                     cpu_limit: 3.725
                     node_selector:
-                      node.kubernetes.io/instance-type: r5.xlarge
+                      node.kubernetes.io/instance-type: r5.4xlarge
                 mem_60_6:
                   display_name: 60.6 GB RAM, upto 15.6 CPUs
```

Comment:
There are gow peofiles not set go 16x, but staying at 4x - but you know they won't use those peofiles during the workshop perhaps?

Reply:
Correct, they had mentioned which ones they're going to use and not these